<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passer</class>
 <widget class="QDialog" name="passer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1101</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/image/wt.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>541</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;

  font-size: 19px;</string>
   </property>
   <property name="title">
    <string>Passer une commande</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
    <item>
     <property name="text">
      <string>-----------------------------</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
    <item>
     <property name="text">
      <string>-----------------------------</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="N_fact_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="N_fact_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Ajouter_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color:rgb(108, 149, 203)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/icons8-add-50.png</normaloff>:/image/icons8-add-50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 10px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color:rgb(108, 149, 203)</string>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix_Unitaire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantite</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ref_Article</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>N°Commande</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>-10</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/image/Capture d’écran 2022-04-06 014158.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pdf">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 8px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 14px;
 
  border-radius : 10px;
background-color:rgb(108, 149, 203)</string>
   </property>
   <property name="text">
    <string>pdf</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/icons8-pdf-50.png</normaloff>:/image/icons8-pdf-50.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="Statistiques">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 8px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 14px;
 
  border-radius : 10px;
background-color:rgb(108, 149, 203)</string>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/icons8-pie-chart-report-50.png</normaloff>:/image/icons8-pie-chart-report-50.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Atualiser">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>620</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 8px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 14px;
 
  border-radius : 10px;
background-color:rgb(108, 149, 203)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/icons8-synchronize-48.png</normaloff>:/image/icons8-synchronize-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="quitter">
   <property name="geometry">
    <rect>
     <x>962</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 8px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 14px;
 
  border-radius : 10px;
background-color:rgb(108, 149, 203)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/icons8-log-out-64.png</normaloff>:/image/icons8-log-out-64.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>390</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00002d;
  font-size: 16px;</string>
   </property>
   <property name="title">
    <string>Supprimer</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Id Commande</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 20px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color:rgb(108, 149, 203)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/icons8-delete-64.png</normaloff>:/image/icons8-delete-64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="num_sup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 7px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color:rgb(108, 149, 203)</string>
   </property>
   <property name="text">
    <string>N° Commande</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>460</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 7px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 16px;
 
  border-radius : 16px;
background-color:rgb(108, 149, 203)</string>
   </property>
   <property name="text">
    <string>Prix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>541</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>541</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recherche_button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>620</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>calibri</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 2px solid black;
border-radius: 10px;
  color: white;
  padding: 10px;
  text-align: center;
  
font-family:calibri;
  display: inline-block;
  font-size: 14px;
 
  border-radius : 10px;
background-color:rgb(108, 149, 203)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/icons8-search-64 (2).png</normaloff>:/image/icons8-search-64 (2).png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>N°Commande</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>580</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref_Article</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
